### PR TITLE
GlobalMarineNet.XSetup version 1.3

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107507)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by upintheairsheep** [comment]
> Automatic Validation ended with: image

That's it. Portable app.

File: manifests/g/GlobalMarineNet/XSetup/1.3/GlobalMarineNet.XSetup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GlobalMarineNet.XSetup
+PackageVersion: 1.3
+Installers:
+- InstallerUrl: http://downloads.globalmarinenet.com/downloads/xsetup1.3.exe
+  Architecture: x86
+  InstallerType: nullsoft
```
Comment (Trenly) [inline manifests/g/GlobalMarineNet/XSetup/1.3/GlobalMarineNet.XSetup.installer.yaml:9]:
```suggestion
  InstallerType: portable
```

**Review by Trenly — CHANGES_REQUESTED** [review]
> That's it. Portable app.

This is why it is important to test manifests before submission. This PR has been open for several days simply because the incorrect installer type was specified. It is usually specified on a publisher's website if an app is a portable app, and if it isn't specified it is very easy to check yourself by downloading and launching the .exe - If it opens a dialogue to install then it is an exe/inno/nullsoft/burn; if it opens the application, then it is portable

**Comment by stephengillie** [comment]
Changing the InstallerType from `nullsoft` to `portable` merely results in the installer being saved and a symlink created. 
image

**Comment by stephengillie** [comment]
> How do I get these images myself?

It's a few steps to navigate there:
1. Click the [Checks](https://github.com/microsoft/winget-pkgs/pull/107507/checks) tab at the top. 
2. Click the `> Azure Pipelines` drop-down and click the WinGetSvc-Validation link
- Alternate: click the [WinGetSvc-Validation](https://github.com/microsoft/winget-pkgs/pull/107507/checks?check_run_id=13738129128) checks link at the bottom of the PR. 
3. Click the link [View more details on Azure Pipelines](https://dev.azure.com/ms/ed6a5dfa-6e7f-413b-842c-8305dd9e89e6/_build/results?buildId=461611)
4. Find the [3 published](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=461611&view=artifacts&pathAsName=false&type=publishedArtifacts) link near the middle of the page. If it says `2 published` then logs weren't generated for the run. 
5. Unfold the `> InstallationVerificationLogs` and download them, or find [Download Artifacts](https://dev.azure.com/ms/ed6a5dfa-6e7f-413b-842c-8305dd9e89e6/_apis/build/builds/461861/artifacts?artifactName=ValidationResult&api-version=7.0&%24format=zip) under the 3-dot menu. 

Or you can use this script I wrote:
```
Function Get-AutoValLogs {
	param(
		$clip = (Get-Clipboard),
		$MainFolder = "C:\temp", #Temp logs storage location, can be anywhere.
		$DestinationPath = "$MainFolder\Installers",
		$LogPath = "$DestinationPath\InstallationVerificationLogs\",
		$ZipPath = "$DestinationPath\InstallationVerificationLogs.zip",
		[switch]$Force
	)
	Get-Process *photosapp* | Stop-Process
	$build = (($clip -split "=")[1])
	start-process "https://dev.azure.com/ms/ed6a5dfa-6e7f-413b-842c-8305dd9e89e6/_apis/build/builds/$build/artifacts?artifactName=InstallationVerificationLogs&api-version=7.0&%24format=zip"
	sleep 2;
	if (!(Test-Path $ZipPath) -AND !$Force) {
		Write-Host "No logs."
		break;
	}
	Remove-Item $LogPath -Recurse -ErrorAction Ignore
	Expand-Archive $ZipPath -DestinationPath $DestinationPath;
	#Get-ChildItem "$MainFolder\Installers\Install*" | Remove-Item -Recurse #Clean out installer logs
	Remove-Item "$DestinationPath\InstallationVerificationLogs.zip"
	(
		(gci $LogPath).FullName| %{
		if ($_ -match "png") {Start-Process $_} 
		Get-Content $_ |Where-Object {
			$_ -match '[[]FAIL[]]' -OR $_ -match 'fail' -OR $_ -match 'error' -OR $_ -match 'exception'}
		}
	) -split "`n" | Select-Object -Unique|clip;
	Get-CannedResponse AutoValEnd
}
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/107507/files#r1204516266

**Comment by upintheairsheep** [comment]
if it installs then automatically launnches, shouldnt this be an interactive installer issue?

**Pull request closed without merging**